dev-guide/ci/events-files.md

```diff
@@ -0,0 +1,66 @@
+# Openshift CI produced files related to events
+
+To aid in debugging, the Openshift CI (prow) produces various files related to events.  The files
+reside in the gather-must-gather and gather-extra subdirectories in the artifacts storage area.
+You will see the "Artifacts" link in the upper right of a prow job (for example, see
+[this prow job](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664).
+
+This document provides details on where the files are located and what they contain.
+
+## Events files
+
+* event-filter.html
+  * included in the output of a `oc adm must-gather`
+  * contains events from namespaces gathered by `oc adm must-gather` plus events that exist in the kube-apiserver at the time `oc adm must-gather` was run.
+    Namespaces not listed as relatedResources for clusteroperators will be missing.
+  * located in the prow job stored artifacts at `.../gather-must-gather/artifacts/` and contained in the must-gather.tar
+    file in that same location
+  * [Example](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/gather-must-gather/artifacts/event-filter.html)
```

Comment:
before we get down to individual step content like this, it's probably worth talking about the `.../{job-name}/{run-id}/artifacts/{job-slug}/{step-name}/artifacts/` structure to get artifacts from each step.  Once we have established that pattern, we can say things like "in gather-must-gather assets" without having to steps to get there or linking to things that will be garbage collected a few months from now.